terraspace/Terrafile

```diff
@@ -0,0 +1,14 @@
+# Starter Terrafile
+# More info: https://terraspace.cloud/docs/terrafile/
```

Comment:
FYI, We most likely will not be making use of this concept.  I can explain more over chat